axis_backend/axis_backend/settings.py

```
@@ -0,0 +1,122 @@
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-@s5lf&37%!v@5bz_5ai0wfdkss2-!3t5xpz9@h)!x6lt4s0825'
```

Ruff (0.14.5)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)